### PR TITLE
Add deprecation warning for truffle

### PR DESCRIPTION
**Pull request by norhh**
shows deprecation warning when using truffle.

### CODE REVIEW

File: mythril/support/truffle.py [hunk 1]
```diff
@@ -181,7 +194,10 @@ def get_mappings(source, deployed_source_map):
         if len(mapping) > 2 and len(mapping[2]) > 0:
             idx = int(mapping[2])
 
-        lineno = source.encode("utf-8")[0:offset].count("\n".encode("utf-8")) + 1
+        if idx == -1:
```
Comment (JoranHonig) [inline mythril/support/truffle.py]: this is not part of this pr right?

Comment (norhh) [reply mythril/support/truffle.py]: Yes, removed it :+1:
